### ISSUE TITLE
Play sound from JS bundle (main.jsbundle)

### ISSUE THREAD

**Issue by gnestor** [zmxv/react-native-sound#14]
As of react-native 0.14 (https://facebook.github.io/react-native/docs/images.html#static-image-resources), images and other assets can be bundled in the JS bundle and used by simply requiring them (e.g. `<Image source={require('./my-icon.png')} />`). Is it possible to play sounds that are in the JS bundle using react-native-sound?

Example:

```javascript
let sound = new Sound('filename', Sound.JS_BUNDLE, (error, props) => {
    sound.play()
})
// or
let sound = new Sound(require('./filename'), (error, props) => {
    sound.play()
})
```

**Comment by zmxv** [comment]
This sounds like a good idea. It will save us the trouble of manually adding sound assets to XCode.
I'm not sure if the require syntax works for non-images, though. Can you confirm?

**Comment by gnestor** [comment]
@amasad says it does: https://github.com/facebook/react-native/issues/1043#issuecomment-97249646

**Comment by danleveille** [comment]
@zmxv @gnestor So should this be working? Using require('./sound.mp3') gives me the following error:

> Unable to resolve module ./sound.mp3 from [project]/index.ios.js: Invalid directory [project]/sound.mp3

**Comment by zmxv** [comment]
@danleveille Where is sound.mp3 located in your project?

**Comment by danleveille** [comment]
@zmxv For now I have it in the base project folder (same folder as index.ios.js, etc.). The path the error outputs is correct, which is strange because the file is definitely there.

**Comment by danleveille** [comment]
Just doing some more debugging:

When I do this:
`var sound = require('./sound.mp3');`

I get the error:
> Unable to resolve module ./sound.mp3 from [project]/index.ios.js: Invalid directory [project]/sound.mp3

However, using an image seems to work fine:

`var image = require('./image.png');`

It seems like mp3 files aren't being included. Even if I put the MP3 file in node-modules/react-native-sound/

Any clue on how to make sure the mp3 file is included?

**Comment by zmxv** [comment]
@danleveille Is the mp3 file listed in the "Copy Bundle Resources" section under "Build Phases"?

**Comment by danleveille** [comment]
@zmxv No. Maybe I misinterpreted this Github issue. I thought @gnestor was suggesting that using require() would automatically include the MP3 file on both iOS and Android without having to manually add the sound files to to Xcode (and an Android directory).

**Comment by mrpatiwi** [comment]
What's the current state of this issue?

**Comment by zmxv** [comment]
@mrpatiwi This is not a high priority feature on my to-do list.
